pkgs/by-name/jo/journald-notify-nu/package.nix

```
@@ -0,0 +1,81 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nushell,
+  makeWrapper,
+  libnotify,
+  systemd,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
This allows better recursion in some places